src/definitions/modules/dropdown.js

```diff
@@ -2391,7 +2397,7 @@
                         $nextSelectedItem
                             .addClass(className.selected)
                         ;
-                        if (settings.selectOnKeydown && module.is.single() && !$nextItem.hasClass(className.actionable)) {
+                        if (settings.selectOnKeydown && module.is.single() && !$nextSelectedItem.hasClass(className.actionable)) {
```

Comment:
did I fixed the name to an intended variable?

Reply:
You actually found a bug and fixed it. 👍🏼 Deserves a separate PR, please.

Reply:
fixed in #2591